manifests/master/base.pp

```diff
@@ -64,7 +75,8 @@
     require    => [
       Package[$::pupmod::master::service],
       Group['puppet'],
-      Class['pupmod::master::sysconfig']
+      Class['pupmod::master::sysconfig'],
+      Puppet_authorization[$auth_conf],
```

Comment:
Should this be a `notify` from `puppet_authorization` or a `subscribe` here?

Reply:
No; I want the authorization to exist before the exec resource is created, but I don't want to trigger the exec on that basis.

Reply:
@pillarsdotnet In this case, a notify from the `puppet_authorization[$auth_conf]` would cause the `puppetserver` service to restart. That's what you would generally want, right?

Reply:
Yeah, except that I don't want the `puppetserver` service to start *before* the `puppet_authorization` is created. A notify doesn't necessarily enforce that ordering; it might cause the service (or exec) to run *twice*.

Reply:
I don't think it means the same thing, but if you insist, I'll try a `notify` after I get things working this way.